src/App.jsx

```diff
@@ -388,29 +392,56 @@ class App extends Component {
       }
   }
 
-  filter() {
+  filter(defaultFilter) {
       var items = Array.from(document.getElementsByClassName("ng-not-empty"));
       var sections = Array.from(document.getElementsByClassName("section"));
       var empty = Array.from(document.getElementsByClassName("ng-empty"));
 
+      let checked, filterCheckbox;
+      if(!defaultFilter) {
+        filterCheckbox = document.getElementById("filterCheckbox");
+        checked = filterCheckbox ? filterCheckbox.checked : false;
```

Comment:
there's a minor bug when doing a standalone launch. The filter checkbox is unchecked, but the form is filtered to begin with.  Clicking the filtered button checks it, but nothing changes, unchecking it unfilters the form and it then works normally.

Making any change other than clicking the filter button, like entering a value in one of the questions, triggers the filter function and the page goes back to being unfiltered.  I think this has to do with the code above, searching specifically for the filter element in the DOM to check its status.  The app seems to be doing its default filter while also maintaining the state as "unfiltered" which indicates a disconnect in React's internal state.

Reply:
@KeeyanGhoreshi I thought standalone launch doesn't open the form as I checked with you last time :-)